src/mlm_client_engine.inc

```diff
@@ -1723,7 +1723,11 @@ int
 mlm_client_set_consumer (mlm_client_t *self, const char *stream, const char *pattern)
 {
     assert (self);
-
+    // https://github.com/zeromq/malamute/issues/217
+    // problem  : if pattern "*" is used, the related stream will be broken and
+    //            mlm broker will start overcosumming 100% CPU forever
```

Comment:
@jimklimov double typo for upstream: overco*n*sum**ing (2nd 'm' suppressed)

Reply:
thanks